include/alpaka/mem/buf/BufUniformCudaHipRt.hpp

```diff
@@ -15,8 +15,10 @@
 // Backend specific includes.
 #    if defined(ALPAKA_ACC_GPU_CUDA_ENABLED)
 #        include <alpaka/core/Cuda.hpp>
+struct ApiCudaRt;
 #    else
 #        include <alpaka/core/Hip.hpp>
+struct ApiHipRt;
 #    endif
```

Comment:
But if both backends are enabled, you would need both includes?
```suggestion
#        include <alpaka/core/Cuda.hpp>
#        include <alpaka/core/Hip.hpp>
struct ApiCudaRt;
struct ApiHipRt;
```

Reply:
Yes.
Also, the forward declarations should be in the `alpaka` namespace - I'll fix them.

Reply:
Well, it's actually not possible to enable both backends at the same time, because of the conflicts between `cuda_runtime.h` and `hip/hip_runtime.h` :-/